gateway-provider-ha/src/main/java/org/apache/knox/gateway/ha/provider/HaServiceConfig.java

```diff
@@ -57,4 +57,8 @@
   boolean isNoFallbackEnabled();
 
   void setNoFallbackEnabled(boolean noFallbackEnabled);
+
+  void setDisableStickySessionForUserAgents(String disableStickySessionForUserAgents);
+
+  String getDisableStickySessionForUserAgents();
```

Comment:
Should return a `List<String>`

Reply:
(minor) nit: The method name is misleading to me. 
List<String> getStickySessionDisabledUserAgents() ?

Reply:
true, will change it.